docs/whatsnew/5.2.rst

```diff
@@ -12,7 +12,25 @@ the 5.1 release.
 
 In particular, this release includes:
 
-* nothing yet
+* :ref:`whatsnew-5.2-cosmology`
+
+
+.. _whatsnew-5.2-cosmology:
+
+Updates to `astropy.cosmology`
+==============================
+
+A new comparison function has been added -- ``cosmology_equal()`` -- that
```

Comment:
There's no reason not to have a hyperlink to the function documentation.

Reply:
SGTM. Habit from the changelog fragments, which aren't meant to be linked.